packages/state-transition/src/cache/epochCache.ts

```diff
@@ -667,10 +667,13 @@ export class EpochCache {
       this.shufflingCache
         .get(epochAfterUpcoming, this.nextDecisionRoot)
         .then((shuffling) => {
-          if (!shuffling) {
-            throw new Error("EpochShuffling not returned from get in afterProcessEpoch");
-          }
-          this.nextShuffling = shuffling;
+          this.nextShuffling = shuffling
```

Comment:
when `asyncShufflingCalculation=undefined/false` I'd like to synchronously compute shuffling so that it works like before and we don't have to worry about anything at syncing time

Reply:
I mean
```typescript
if (!asyncShufflingCalculation) {
 // sync code, work like before
  this.nextShuffling = ...
} else {
// async flow handled through .then
  this.shufflingCache
        .get(epochAfterUpcoming, this.nextDecisionRoot)
        .then((shuffling) => {
}
```

Reply:
ok. updated